glide.lock

```diff
@@ -385,14 +385,16 @@ imports:
   version: 449fdfce4d962303d702fec724ef0ad181c92528
   subpackages:
   - spdy
+- name: github.com/dustin/go-humanize
+  version: bb3d318650d48840a39aa21a027c6630e198e626
 - name: github.com/elazarl/go-bindata-assetfs
   version: 3dcc96556217539f50599357fb481ac0dc7439b9
 - name: github.com/emicklei/go-restful
   version: ff4f55a206334ef123e4f79bbf348980da81ca46
   subpackages:
   - log
 - name: github.com/emicklei/go-restful-swagger12
-  version: 5e28dc7157ebd9247a268c26dd1bee7818aa33f7
+  version: 14b2c9246e6851e0297450a3d5c7288c0485592e
```

Comment:
where is this SHA from? this looks like it is moving us off our branch: https://github.com/openshift/emicklei-go-restful-swagger12/commits/openshift-3.9

Reply:
@liggitt hrm weird, I thought this is transitive etcd dependency. Have we pinned this in glide?